task_manager/models/task.py

```diff
@@ -1,18 +1,22 @@
-from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Text
+from sqlalchemy import Column, ForeignKey, Integer, String, Text
 from task_manager import db, ma
+from task_manager.models.user import User
+# User imported for db.relationship
 
 
 class Task(db.Model):
     __tablename__ = 'task'
     id = Column(Integer, primary_key=True)
     title = Column(String(80), nullable=False)
-    priority = Column(Integer, default=1, nullable=False)  # do a check to make sure is in range
+    priority = Column(Integer, default=1, nullable=False)
+    # do a check to make sure is in range
     description = Column(Text, nullable=True)
-    start_time = Column(DateTime, nullable=True)
-    end_time = Column(DateTime, nullable=True)  # make sure end time is after start time
+    start_time = Column(String(80), nullable=True)
```

Comment:
Why change datetime to string? I think that might make it hard to check whether it is a valid datetime

Reply:
Its not as user friendly